src/main/java/steps/TestResultsSteps.java

```diff
@@ -352,4 +329,23 @@ public void validateTestCode(TestResultsGet data, String... expectedTestCodes) {
         }
 
     }
+
+    @Step
+    public void validateTestNumberIsNotNull() {
+        assertThat(validateTestNumber()).isTrue();
```

Comment:
why assertThat instead of response.then() ?

Reply:
This the way I thought it was a better way to handle null assertions.